apps/dev/src/main.scss

```diff
@@ -1,4 +1,5 @@
 // from "* { Box-sizing: Border-box } FTW" @ https://www.paulirish.com/2012/box-sizing-border-box-ftw/
+@import '~@dynatrace/barista-fonts/typography';
```

Comment:
```suggestion
```

remove that you are already importing that in the [index.scss](https://github.com/dynatrace-oss/barista/blob/master/components/style/index.scss#L7).

And in the dev app you include the `index.scss` with the styling.